cdap-docs/developers-manual/source/data-exploration/filesets.rst

```diff
@@ -43,8 +44,12 @@ For example, in the configure method of your application::
       .setTableProperty("avro.schema.literal", schema.toString())
       .build());
 
-These dataset properties map directly to table properties in Hive. 
-For example, the dataset above would result in the following "create table" statement being generated::
+These dataset properties map directly to table properties in Hive. In the case of the
+``setBasePath`` method, the partial-path given will be a sub-directory of
+``<CDAP-home>/namespaces/<current-namespace>/data/``.
```

Comment:
sorry  about missing this earlier. Is ``current-namespace`` the right term here?

I don't think we have a notion of 'current' namespace, unless you are in the UI? 

What is the pattern in the rest of the docs? Do they always use default namespace?